docs/source/releases.rst

```diff
@@ -1,6 +1,112 @@
 Release History
 ###############
 
+
+v8.5.0 (2024-08-22)
+===================
+
+Features
```

Comment:
I'm considering whether this should be changed from "Features" to "Library features" (new components, base classes, utils, etc.) to make it more clear that the device-specific features go in "Device Updates"